lib/matplotlib/tests/test_texmanager.py

```diff
@@ -57,3 +60,15 @@ def test_unicode_characters():
     with pytest.raises(RuntimeError):
         ax.set_title('\N{SNOWMAN}')
         fig.canvas.draw()
+
+
+@needs_usetex
+def test_openin_any_paranoid():
+    completed = subprocess.run(
+        [sys.executable, "-c",
+         'import matplotlib.pyplot as plt;'
+         'plt.rcParams.update({"text.usetex": True});'
+         'plt.plot("paranoid");'
```

Comment:
```suggestion
         'plt.title("paranoid");'
```
While it works, `plot("paranoid")` is quite a strange plot (one data point as x=0 and categorical y="paranoid"). Let's use a simpler and more standard test plot: an emptry plot with "paranoid" as title.

Reply:
You're right, thanks, I wasn't really thinking about what I was actually plotting here.